com.github.dawidd6.checksumo.yml

```diff
@@ -0,0 +1,23 @@
+app-id: com.github.dawidd6.checksumo
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: checksumo
+finish-args:
+  - --filesystem=host
```

Comment:
Would it be difficult to use GtkFileChooserNative for opening files? This way the app wouldn't need this sandbox hole.

Reply:
To be honest I was surprised that the app had access to the whole filesystem even without `--filesystem=host` option. I just added it, cause I see other manifests have it declared too. Is it possible GTK already makes use of portals via GtkFileChooserButton? If not, why did it work before?

Reply:
GtkFileChooserButton Is a wrapper around a GtkFileChooser, not a native one. You can do the same thing manually though by creating a Button & opening a native file chooser when clicked on.

Reply:
I decided to take a look at the GTK code and found this:

https://github.com/GNOME/gtk/blob/a02de1c9d9ed99e51ae4470c28644d3222a0bf8f/gtk/gtkfilechooserbutton.c#L784-L796

Judging by it, GtkFileChooserButton indeed uses GtkFileChooserNative if dialog wasn't explicitly set (which I do not do in my code).

So now I know why `--filesystem` option wasn't needed from the start. Imma delete it.